.github/workflows/ci.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: [main]
     tags: ["v*"]
-  pull_request
+  pull_request:
```

Comment:
Is it `github actions NOT printing an explicit error in case of malformed config file`, vol. 2137? :trollface: